src/commands/createFunctionApp/containerImage/ContainerizedFunctionAppCreateStep.ts

```diff
@@ -84,6 +74,23 @@ export class ContainerizedFunctionAppCreateStep extends AzureWizardExecuteStep<I
             }
         }
     }
+
+    protected getTreeItemLabel(context: IFunctionAppWizardContext): string {
+        const siteName: string = nonNullProp(context, 'newSiteName');
+        return localize('creatingNewApp', 'Create new function app "{0}"', siteName);
```

Comment:
Same nit as above

Reply:
Matt be like:

<img src="https://media3.giphy.com/media/v1.Y2lkPWJkM2VhNTdleG5vcmU4b2hla2RldmU2ZnR0cjRtc3NqcGttbXB1cWIxNXFyYmw0aiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/jUnGHmsHdzkxbB63Mf/giphy.gif"/>